cmd/tempo/app/modules.go

```diff
@@ -290,6 +301,7 @@ func (t *App) setupModuleManager() error {
 	mm.RegisterModule(Compactor, t.initCompactor)
 	mm.RegisterModule(Store, t.initStore, modules.UserInvisibleModule)
 	mm.RegisterModule(All, nil)
+	mm.RegisterModule(HA, t.initHA)
```

Comment:
why not follow the same pattern as `All` and pass nil here? Is there a need for the init function?